spec/integration/games_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "spec_helper"
+describe "a new game", type: :feature do
+
+  describe "visiting the new game page" do
+    include Capybara::DSL
+
+    it "has the link to the answer word's sound clip" do
+      user = create(:user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).
+        and_return(user)
+      8.times { create(:upload) }
+
+      visit new_game_path
+
+      expect(page.find("#game-clip")['src']).
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.